#
# Oracle Solaris specific patch. We don't do ident lookups by default
#

--- a/doc/Configuration.html
+++ b/doc/Configuration.html
@@ -10805,7 +10805,7 @@ CLASS="SYNOPSIS"
 ></DT
 ><DD
 ><P
->IdentLookups on</P
+>IdentLookups off</P
 ></DD
 ><DT
 ><PRE
